### DISCUSSION TITLE
Multi-Channel Chat view.

### DISCUSSION THREAD

**Discussion by LividJava** [Chatterino/chatterino2#4903] [Ideas] [answered]
A Multi-Channel chat view where you can have more than one channel in a view.

Similar to how Sheepchat implements its cross-platform chat. Allow for multiple twitch channels in one panel. I understand the limitations this may have when it comes to actually sending messages, but as a streamer I only need it to be read-only. Meaning you would not be able to send messages in this panel, just read them.

This would allow me to keep up with more than one chat during a collaborative stream as though it were just one panel. Rather than needing to have 4+ panels to split my attention between.

**Comment by Felanbird** [comment]
This is already technically possible with the mentions tab, just add a highlight that matches every message (e.g. `.` with regex enabled) and only look at the `/mentions` tab.
![image](https://github.com/Chatterino/chatterino2/assets/41973452/ae7d44b8-9378-46c7-a0e8-c3efac98ff47)

**Comment by LividJava** [comment]
@Felanbird I'll be honest, while that does work. it does require me to ONLY have the channels i want to see open. So I cannot have dedicated views or what have you.

I would much rather like to see this as a dedicated feature versus this solution.

**Reply by Felanbird** [reply, marked as answer]
I gave you the idea because otherwise this request is just a dupe of #4056 - but read only instead of their idea of both read and write.

**Reply by LividJava** [reply]
Ah, i didn't see that one when searching around Apologies for that.

**Comment by LividJava** [comment]
> I gave you the idea because otherwise this request is just a dupe of #4056 - but read only instead of their idea of both read and write.
